SNOW-1256835: Fix snowflake.yml to match readme.md steps (#6)
Snowflake-Labs/sfguide-native-apps-chairlift: README.md

|Chairlift sensors|A record of all sensors configured on these machines|Choose `chairlift_consumer_data.data.sensors`|
|Chairlift sensor readings|Data recorded from the above sensors, generated by `chairlift_consumer.data.populate_reading`|Choose `chairlift_consumer_data.data.sensor_readings`|

> Once the references have been declared in the application manifest, consumers can grant the privilege to `EXECUTE TASK` to the application by clicking the `Request Privileges` button.

Reviewer: I didn't realize the context of this line -- here's my newer suggestion
```suggestion
Because the privilege has been declared in the application manifest, consumers can grant `EXECUTE TASK` to the application by clicking the `Request Privileges` button when it comes up.
```
